sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -220,6 +223,8 @@ public String getEntityPath() {
      *     {@code transactionContext.transactionId} is null when {@code options.transactionContext} is specified.
      * @throws UnsupportedOperationException if the receiver was opened in
      *     {@link ServiceBusReceiveMode#RECEIVE_AND_DELETE} mode or if the message was received from peekMessage.
+     * @throws IllegalStateException if receiver is already disposed.
+     * @throws IllegalArgumentException if message is already been settled.
```

Comment:
Make the changes in other places too.

```suggestion
     * @throws IllegalArgumentException if the message is already settled.
```